tests/unit/test_models.py

```diff
@@ -846,6 +848,139 @@ def test_contains_container_types(container_column, engine):
     assert renderer.rendered == expected
 
 
+def test_column_defaults():
+
+    # Set should send the new value
+    called = False
+    num_called = 0
+    received = {}
+
+    @object_modified.connect
+    def verify_expected(_, *, obj, column, value):
+        nonlocal num_called, called, received
+        called = True
```

Comment:
nit: `num_called` provides a strict superset of the information of `called`, so you can remove `called`.

Reply:
Good call.